pkg/api/resource/quantity.go

```diff
@@ -184,14 +193,13 @@ func ParseQuantity(str string) (*Quantity, error) {
 	if sign == -1 {
 		amount.Neg(amount)
 	}
-	// This rounds non-zero values up to the minimum representable
-	// value, under the theory that if you want some resources, you
-	// should get some resources, even if you asked for way too small
-	// of an amount.
-	// Arguably, this should be inf.RoundHalfUp (normal rounding), but
-	// that would have the side effect of rounding values < .5m to zero.
+
+	// This rounds non-zero values up to the minimum representable value, under the theory that
+	// if you want some resources, you should get some resources, even if you asked for way too small
+	// of an amount.  Arguably, this should be inf.RoundHalfUp (normal rounding), but that would have
+	// the side effect of rounding values < .5n to zero.
 	if v, ok := amount.Unscaled(); v != int64(0) || !ok {
-		amount.Round(amount, 3, inf.RoundUp)
+		amount.Round(amount, 9, inf.RoundUp)
```

Comment:
Fixed magic number.

> does this mean I can ask for a nano-core now?

I don't think this should be determined by the unit. If we want to say the smallest schedulable unit is a milli-core, then I think we should document that in the API and enforce it in the validation step (or just round up in the conversion?).